react-cardknox-ifields/index.d.ts

```diff
@@ -1,3 +1,4 @@
+import { string } from "prop-types"
```

Comment:
Is this needed? It looks like 'string' was already being used

Reply:
Removed